frontend/src/metabase/visualizations/components/ChartSettings.jsx

```diff
@@ -227,7 +227,7 @@ class ChartSettings extends Component {
 
     // default layout with visualization
     return (
-      <div>
+      <div className="flex flex-column spread">
```

Comment:
`ChartSettings` is used in `DashCard.jsx` (`ChartSettingsWithState`) and `ChartSettingsSidebar.jsx`. I'm not sure `spread` should be applied to both, but even if it's fine in this case generally I avoid putting classes that affect the layout within the parent on the root element of a component:

> In order to make components more reusable, a component should only apply classes or styles to the root element of the component which affects the layout/styling of it's own content, but not the layout of itself within it's parent container. For example, it can include padding or the flex class, but it shouldn't include margin or flex-full, full, absolute, spread, etc. Those should be passed via className or style props by the consumer of the component, which knows how the component should be positioned within itself.

https://github.com/metabase/metabase/wiki/Frontend:-Style-Guide

I'd change this to `<div className={cx(className, "flex flex-column")}>` and add `className="spread"` to `ChartSettingsWithState` in `DashCard.jsx`

Reply:
Also you should confirm `ChartSettingsSidebar` still looks correct.